UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs

```diff
@@ -108,15 +112,18 @@ public void RunOreGenerator()
 
 	private void NodeScatter(Vector3Int location, OreProbability materialSpecified)
 	{
-		var locations = new List<Vector3Int>() {
+		var locations = new List<Vector3Int>()
+		{
 			location,
 		};
-		var strength = materialSpecified.PossibleClusterSizes[RANDOM.Next(materialSpecified.PossibleClusterSizes.Count)];
+		var strength =
+			materialSpecified.PossibleClusterSizes[RANDOM.Next(materialSpecified.PossibleClusterSizes.Count)];
 		while (strength > 0)
 		{
 			var chosenLocation = locations[RANDOM.Next(locations.Count)];
 			var ranLocation = chosenLocation + DIRECTIONS[RANDOM.Next(DIRECTIONS.Count)];
-			var tile = MetaTileMap.GetTile(ranLocation);
+			var tile = MetaTileMap.GetTile(ranLocation, ignoreEffectsLayer: true);
```

Comment:
does this fix that one bug where some ores are spawned on the effects/overlay layer?

Reply:
Not too sure, This was throwing up errors,  because it was trying to interact with overly tiles